### ISSUE TITLE
[TestUI] Feature Request: Double-clicking a test in the UI pane should "Go To Test" (same behavior as alt-enter)

### ISSUE THREAD

**Issue by JustinGrote** [microsoft/vscode#131456]
@connor4312 

![image](https://user-images.githubusercontent.com/15258962/130537504-933623b4-d74b-43de-b509-f3088def9f53.png)

This should be true for both discovered (but not run) and run tests. 

Currently if a test has an error, single-click will go straight to the error. Perhaps also make this behavior configurable to double-click, as people may not like an unrequested code jump.

Probably best have a setting for test navigation behavior. Options should be:
1. Single Click - Single click on anything does a Go to Test
2. Double Click - Double click on anything (including errors) does a go to test
3. Single Click Error -Single click an error goes to it, everything else requires double click
4. None - Must click the go to test button or alt-enter to navigate

**Comment by Trass3r** [comment]
Indeed, I fixed this before in test explorer but had to use a hack to detect the double-click: https://github.com/hbenl/vscode-test-explorer/pull/161/files
Hopefully this can be solved more cleanly in vscode itself.
